server/routes/index.js

```diff
@@ -61,6 +61,30 @@ module.exports = function Index ({ authenticationMiddleware }) {
     res.render('privacy-notice', { params: { nonce: nonce, backLink: session.backLink } })
   })
 
+  router.get('/cookies-policy', (req, res) => {
+    res.render('cookies-policy', { params: { saved: req.query.saved, preference: req.cookies.analyticsCookies, nonce: nonce } })
```

Comment:
The issue causing the 500 server error is that during testing, `req.cookies` will be undefined and then `req.cookies.analyticsCookies` will throw an error.

Changing this to `req.cookies && req.cookies.analyticsCookies` will ensure that it's checked before accessing and will remove the error.